neo_force_scheme/neo_force_scheme.py

```diff
@@ -164,10 +164,20 @@ def _fit(self, X,
 
 
     def _transform(self, X, *, index, total, inplace, n_dimension: Optional[int] = 2, force_projection_dimensions=None,
-                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0)):
+                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0),
+                   confidence_interval: Optional[float] = 1):
         # iterate until max_it or if the error does not change more than the tolerance
         error = math.inf
 
+        z_score = 1
+        if confidence_interval < 1.0:
```

Comment:
is this the only place confidence_interval is used? is it just an if to select whether its the strict bins or the gaussian?

Reply:
Yes, this is the only place that condifence_interval is used, it is meant to be used to both select the mode and calculate the z_score (which determine the shape of gaussian function)

Reply:
I think then it is good to include what the value of condifence_interval means/is used to in the :params comment. Can you do that?

Reply:
Comment added at neo_force_scheme.py line 330 - 340